nixos/modules/services/continuous-integration/buildkite-agent.nix

```diff
@@ -21,9 +16,11 @@ in
       enable = mkEnableOption "buildkite-agent";
 
       token = mkOption {
-        type = types.str;
+        type = types.either types.str types.path;
```

Comment:
I think we should break backwards compatibility here, since the old method was insecure.

We should just allow `types.path`.

Reply:
Done.